engine/src/main/java/org/terasology/rendering/world/WorldRendererLwjgl.java

```diff
@@ -78,57 +76,37 @@
 import static org.lwjgl.opengl.GL11.glPushMatrix;
 
 /**
- * The world of Terasology. At its most basic the world contains chunks (consisting of a fixed amount of blocks)
- * and the player.
- * <p/>
- * The world is randomly generated by using a bunch of Perlin noise generators initialized
- * with a favored seed value.
- *
  * @author Benjamin Glatzel <benjamin.glatzel@me.com>
  */
 public final class WorldRendererLwjgl implements WorldRenderer {
-    public static final int VERTICAL_SEGMENTS = CoreRegistry.get(Config.class).getSystem().getVerticalChunkMeshSegments();
 
     private static final int SHADOW_FRUSTUM_BOUNDS = 500;
 
-    /* WORLD PROVIDER */
+    private final int verticalMeshSegments = CoreRegistry.get(Config.class).getSystem().getVerticalChunkMeshSegments();
+
+    private final Skysphere skysphere;
     private final WorldProvider worldProvider;
+    private final RenderableWorld renderableWorld;
 
-    /* PLAYER */
     private LocalPlayer player;
 
-    /* CAMERA */
-    private Camera activeCamera;
+    private final Camera activeCamera;
```

Comment:
In short: you are right. 

Right now I don't think the code can handle switching cameras. For one there is no setActiveCamera() method yet: the camera is created in the constructor and stay the same. So, I just formalized what I think it's the state of affair. Also, RenderableWorld has a pregenerateChunk() method that is only called on world loading time. The method RenderableWorldImpl.updateChunkInProximity() might be enough to regenerate the world around a new camera, but I need to verify that. For sure, RenderableWorld currently has no facility to switch cameras.

That been said, it is something I'd like to sort out. I just need to understand better what needs to be updated when the camera changes. For the time being I can:

1. rename the camera to something like "playerCamera"
2. remove the final keyword
3. add a TODO and leave it as it is, to be worked on in one of the next PRs

What do you prefer?